main/zoe/guide/contracts/multipoolAutoswap.md

```diff
@@ -74,13 +74,13 @@ back.
 
 ```js
 const saraProposal = harden({
-  want: { Out: moolaAmountMath.make(275) },
-  give: { In: simoleanAmountMath.make(220) },
+  want: { Out: amountMath.make(moola, 275n) },
+  give: { In: amountMath.make(simolean, 220n) },
```

Comment:
These are nearly always `moolaBrandd` and `simoleansBrand`. In fact, I see  `simoleansBrand` above.  The issuer name on its own doesn't obviously represent the brand or the issuer, so we don't often use it alone.